README.md

```diff
@@ -2,20 +2,49 @@
 This section describes tools that help you with coding. We provide scripts that allows to
 clone all repositories and install artifacts to Maven local / Maven Central Snapshot repositories.
 
+## Prerequisites on Windows
+Install and configure [Windows Subsystem for Linux](https://docs.microsoft.com/en-us/windows/wsl/install-win10) (WSL) and configure:
+- Java 8 on WSL
+- Gradle 5+
+- Maven
+
 ## Clone all repositories
 Check a `development/pull-all.sh` script to clone all Knot.x repositories. Please check `-h` option
 for help. Execute this script from `knotx-aggregator/development` directory, executing it from another 
-directory will fail.
+directory will fail. Please check `-r` option for providing custom directory where Knot.x repositories 
+will be cloned, for example `sh pull-all.sh -r ../../knotx` make sure, that provided directory exists. 
 
 ## Build Stack
 Check a `development/build-stack.sh` script to build all cloned repositories. Please check `-h` option
-for help.
+for help. This command should also be executed from `knotx-aggregator/development` directory, in other 
```

Comment:
Execute this command from the `knots-aggregator/development` directory. Specify the `-r` parameter to point the directory with cloned repositories:
```
sh build-stack.sh -r ../../knotx
```

Reply:
fixed